### PR TITLE
Add links to ranking pages from stat var chips

### PR DESCRIPTION
**Pull request by beets**
Also carry over params so the per capita, unit and scaling match the original charts.

image

### CODE REVIEW

**Comment by beets** [comment]
This PR fixes quite a few issues at once, so it might be easier to review a commit at a time.

File: static/js/place/chart_block.tsx [hunk 1]
```diff
@@ -70,6 +72,10 @@ class ChartBlock extends React.Component<ChartBlockPropType, unknown> {
     // Plot trend data in overview and topic page.
     if (!_.isEmpty(this.props.data.trend)) {
       const id = randDomId();
+      const rankingParam = new URLSearchParams(`h=${this.props.dcid}`);
+      this.props.data.denominator && rankingParam.set("pc", "1");
```
Comment (shifucun) [inline static/js/place/chart_block.tsx:76]: this has no efffect?

Comment (beets) [reply static/js/place/chart_block.tsx:76]: it adds pc=1 to rankingParam, only if this.props.data has a denominator. it was getting long with all the if statements, so i used shorthand -- i can unroll it tho

File: static/js/place/place.ts [hunk 2]
```diff
@@ -178,10 +178,17 @@ function renderPage(dcid: string) {
     updatePageLayoutState();
 
     // Display child places alphabetically
+    let childPlaceType: string;
+    let childPlaceTypeCount = 0;
     for (const placeType in data.allChildPlaces) {
       data.allChildPlaces[placeType].sort((a, b) =>
         a.name < b.name ? -1 : a.name > b.name ? 1 : 0
       );
+      const numChildren = data.allChildPlaces[placeType].length;
```
[the remark this answers is not in the document]
Comment (beets) [reply static/js/place/place.ts:187]: ok, i'll add it to the mixer and add a todo to remove this logic after that is added

[the remark this answers is not in the document]
Comment (shifucun) [reply static/js/place/place.ts:187]: sg.

**Review by shifucun — APPROVED** [review]

**Review by shifucun — APPROVED** [review]

**Pull request merged**